src/plugins/login/loginPlugin.test.ts

```diff
@@ -0,0 +1,123 @@
+import { MockFactory } from "../../test/mockFactory";
+import { invokeHook } from "../../test/utils";
+import { AzureLoginPlugin } from "./loginPlugin";
+import { AzureLoginService } from "../../services/loginService";
+
+describe('Login Plugin', () => {
+
+  const authResponse = MockFactory.createTestAuthResponse();
```

Comment:
lots of duplicate code for setting up AzureLoginService.  can we move some of them to a `beforeEach`?